<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWidgetEditor</class>
 <widget class="QWidget" name="EmployeeWidgetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font-size: 13px;
}</string>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSurname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMiddleName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFunction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSalary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарплата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>9</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSurname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMiddleName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFunction">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSalary">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
